### PR TITLE
launch config examples fixes

### LINKED ISSUE #1862: Launch config BFV example is incorrect
2nd example in https://developer.rackspace.com/docs/autoscale/v1/developer-guide/#launch-configuration-examples is incorrect.

### PR DESCRIPTION
**Pull request by manishtomar**
- corrected BFV example. Fixes #1862 
- better formatted
- removed reference about base64 encoded personality since that example
  is not used anymore

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/build-412ab6b3c0/docs/autoscale/v1/developer-guide/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-24e8d6a0c9/docs/autoscale/v1/developer-guide/).

**Comment by manishtomar** [comment]
The preview looks good :) @meker12 Could you review this please?

**Comment by meker12** [comment]
https://staging.developer.rackspace.com/build-412ab6b3c0/docs/autoscale/v1/developer-guide/#document-concepts -- Check again?
rackspace_autoscale_1_0_-_rackspace_developer_portal

File: api-docs/rst/dev-guide/concepts.rst [hunk 1]
```diff
@@ -236,96 +236,79 @@ The following example shows a launch configuration of type
...
-              },
-              {
-                "uuid": "00000000-0000-0000-0000-000000000000"
-              }
+            "loadBalancers": [
+                {
+                    "loadBalancerId": 9099,
+                    "port": 8080
+                }
             ],
```
Comment (meker12) [inline api-docs/rst/dev-guide/concepts.rst]: Get rid of blank lines. Remove the "sh" from the code samples.

**Comment by meker12** [comment]
LGTM

**Pull request merged**